ui/src/components/visit-us/VisitUsPage.js

```diff
@@ -175,7 +175,7 @@ const VisitUsPage = (props) => {
               textAlign={['center', 'right']}
             >
               <Link href="https://hongkong.hmcc.net/online/" isExternal>
-                Join us Online <Icon as={RiArrowRightLine} />
+                Join us ONLINE
               </Link>
             </Heading>
           </VStack>
```

Comment:
also, i think the FAQ in the figma is different. can you double check if we're supposed to change the FAQs too?